index.html

```diff
@@ -193,20 +201,47 @@ <h2>About <br> Myself </h2>
         </div>
         <div class="framework">
             <ul>
-                <li><h2>Frameworks</h2></li>
+                <li>
+                    <h2>Frameworks</h2>
+                </li>
                 <li><a href="#"><img src="./images/arrowside.jpg" alt="arrow-side"></a></li>
             </ul>
         </div>
         <div class="framework">
             <ul>
-                <li><h2>Skills</h2></li>
+                <li>
+                    <h2>Skills</h2>
+                </li>
                 <li><a href="#"><img src="./images/arrowside.jpg" alt="arrow-side"></a></li>
             </ul>
         </div>
-        
+
     </div>
+    <!-- footer section -->
 
-    
-        
+    <footer>
+        <div class="contact-me">
+            <h2>Contact Me</h2>
+        </div>
+        <div class="contact-para">
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        </div>
+        <form action="https://formspree.io/f/xpzbddky" method="POST" class="contact-form">
+            <div>
+                <input type="text" name="full-name" id="name" size="40" maxlength="30" placeholder="Full Name" required>
+            </div>
+            <div>
+                <input type="email" name="email" id="email" size="40" maxlength="30" placeholder="Email" required>
+            </div>
+            <div>
+                <textarea name="description" id="description" rows="3" cols="42" maxlength="30"
+                    placeholder="Write your message here" required></textarea>
+            </div>
```

Comment:
- [ ] [REQUIRED] Please note that there is a white space surrounding the form which is not required. You have a margin on the body.
Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![Screenshot from 2022-02-16 17-22-45](https://user-images.githubusercontent.com/58692590/154310832-61f64b02-a366-47cf-91a8-d1b1db2b1abc.png) | ![Screenshot from 2022-02-16 17-22-34](https://user-images.githubusercontent.com/58692590/154310894-3a5718ad-d978-4efc-870c-c8fe1910976c.png)

Reply:
@Whoistolu 
Hi there,
thank you for mentioning those. 
could you please help me regarding the width?
Which property should I use to fix this issue?
Looking forward to hearing from you.